weaviate/data/crud_data.py

```diff
@@ -248,11 +248,11 @@ def update(self,
         raise UnexpectedStatusCodeException("Update of the object not successful", response)
 
     def replace(self,
-            data_object: Union[dict, str],
-            class_name: str,
-            uuid: Union[str, uuid_lib.UUID],
-            vector: Sequence=None
-        ) -> None:
+                data_object: Union[dict, str],
+                class_name: str,
+                uuid: Union[str, uuid_lib.UUID],
+                vector: Sequence = None
+                ) -> None:
```

Comment:
I do not like this indentation, I already used 2 tabs, this change now is 3 tabs, which is one too many. 

Also when an argument has a default value, there are no spaces around the `=`, only when a variable is assigned (this is the style I use).